Updated the ModalForm.jsx

leovaldez08 opened the pull request:
Updated the ModalForm by breaking down the steps into two separate components and imported it. Also made some minor changes in Footer.jsx  and README.md

leovaldez08 committed:
Updated the ModalForm.jsx

Updated the ModalForm by breaking down the steps into two separate components and imported it.

jithin-kalvian commented on FrontEnd/src/Components/Description.jsx:
```diff
@@ -0,0 +1,148 @@
+          color="#333333"
+          bg="white"
+          borderRadius="4px"
+          h="120px"
```
For your width, margin, height properties etc., you've used a mix of percentages, pixel values, and the chakra unit-less numbers (which default to pixels). To maintain consistency and allow for easier theme scaling, you might want to stick to one unit or follow a set pattern for when to use which unit.

leovaldez08 replied:
Noted

jithin-kalvian commented on FrontEnd/src/Components/Details.jsx:
```diff
@@ -0,0 +1,107 @@
+            bg="#6C63FF"
+            color="white"
+            borderRadius="3px"
+            onClick={handleSubmit}
```
This component lacks the logic for disabling the Submit button when the form isn't properly filled. Please add it as per the previous page.

leovaldez08 replied:
Yeah working on it, once I add formik for validation, it'll be resolved!

jithin-kalvian commented on FrontEnd/src/Components/ModalForm.jsx:
```diff
@@ -24,31 +18,43 @@ function ModalForm({ onClose }) {
+  const handleSubmit = useCallback(
+    (event) => {
+      event.preventDefault();
+      console.log(formValues);
```
Please remove the console logs

leovaldez08 replied:
Got it!

jithin-kalvian reviewed (changes requested):
Please complete the requested changes

yash0212 commented on FrontEnd/src/Components/ModalForm.jsx:
```diff
@@ -90,162 +95,22 @@ function ModalForm({ onClose }) {
-                    </Button>
-                  </Stack>
-                </Box>
+                <Form2
```
Please use descriptive names for components

leovaldez08 replied:
Got it, will make changes!

yash0212 commented on FrontEnd/src/Components/ModalForm.jsx:
```diff
@@ -90,162 +95,22 @@ function ModalForm({ onClose }) {
-                    </Box>
-                  </Flex>
-                </Box>
+                <Form1
```
Please use descriptive names for custom components

leovaldez08 replied:
Sure

leovaldez08 committed:
Added react dropzone and formik

Added dropzone for image upload functionality and formik for the modal form validation

leovaldez08 committed:
Updated NavBar and HomePage

Minor styling changes for the buttons

leovaldez08 committed:
Added auth0 for authentication

Installed auth0 authentication and attached event listener to the button in NavBar component.

leovaldez08 committed:
Revert "Added auth0 for authentication"

This reverts commit 28c2f56bc0379b061398a4db901af8d09754b796.
